js_02/ajax.js

```diff
@@ -0,0 +1,32 @@
+function send(){

```

Comment:
You may want to enable the js validations now that the core of the task is done in order to prevent invalid data being sent to the backend.